starter-code/src/viking.js

```diff
@@ -1,11 +1,96 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health -= damage;
+  // return this.health;
+};
 
 // Viking
-function Viking() {}
+function Viking(name, health, strenght) {
+  Soldier.call(this, health, strenght);
+  this.name = name;
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
```

Comment:
Don't forget setting the constructor too `Viking.prototype.contstructor = Viking`